src/core/viewports/orbit-viewport.js

```diff
@@ -1,4 +1,5 @@
 import Viewport from './viewport';
+// import log from '../utils/log';
```

Comment:
do we still need it?

Reply:
The idea here is to issue a deprecation warning when using `OrbitViewport` instead of `OrbitView`, just like for all the other `Viewports` - but currently commented since the OrbitControl has some complexities that need extra consideration.